src/components/templates/AnimateMap/components/ControlPanel/ControlPanel.scss

```diff
@@ -0,0 +1,60 @@
+.ControlPanel {
+  height: 100%;
+  width: 100%;
+
+  &__icon-panel {
+    height: 100%;
+    width: 100%;
+    display: flex;
+    flex-flow: column nowrap;
+    justify-content: space-around;
+    align-items: center;
+  }
+}
+
+.icon-panel {
+  border-radius: 30px;
+  background: rgba(0, 0, 0, 0.15);
```

Comment:
Glenn said you were using BEM. So I broke it like that. 
For the same reason, I have named modifiers (active,disable) "_" rather than "--".

Positioning is performed over an element within a block. In addition, the element, which is also a block, has its own peculiarities - rounded edges and a background color.